tests/posix/fs/src/test_fs_stat.c

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright (c) 2023 Nikhef
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <string.h>
+#include <fcntl.h>
```

Comment:
Please also add the `<zephyr/posix/..>` prefix here

Reply:
Fixed